src/Uno.UI/UI/Xaml/Controls/WebView/WebView.Android.cs

```diff
@@ -25,8 +25,8 @@ namespace Windows.UI.Xaml.Controls
 	public partial class WebView : Control
 	{
 		private Android.Webkit.WebView _webView;
-
-		protected override void OnApplyTemplate()
+        private bool _afterToString;
```

Comment:
I'd rename this to `_wasLoadedFromString`, it's a bit more clear.

Reply:
Is this a must? :)

Reply:
Readability is a must, yes. It can be a different name if you don't like my suggestion, you can add a comment explaining what it does, but it needs to be more clear than it is what that field is doing. Readability is just as important as correctness.